packages/fxa-payments-server/src/images/error.svg

```diff
@@ -0,0 +1,4 @@
+<svg width="80" height="80" viewBox="0 0 80 80" fill="none" xmlns="http://www.w3.org/2000/svg">
+<rect width="80" height="80" rx="40" fill="#FF4F5E"/>
+<path d="M56.7323 50.0297L44.3498 25.2647C43.9347 24.4339 43.2963 23.7353 42.5064 23.247C41.7164 22.7587 40.806 22.5 39.8773 22.5C38.9487 22.5 38.0383 22.7587 37.2483 23.247C36.4584 23.7353 35.82 24.4339 35.4048 25.2647L23.0223 50.0397C22.6437 50.8016 22.4658 51.6475 22.5054 52.4974C22.545 53.3473 22.8008 54.173 23.2486 54.8964C23.6965 55.6198 24.3215 56.217 25.0645 56.6314C25.8075 57.0458 26.6441 57.2638 27.4948 57.2647H52.2573C53.1097 57.2651 53.9481 57.0475 54.6927 56.6328C55.4374 56.218 56.0636 55.6197 56.512 54.8948C56.9604 54.1699 57.216 53.3424 57.2546 52.4909C57.2931 51.6394 57.1134 50.7921 56.7323 50.0297ZM37.3773 32.2647C37.3773 31.6016 37.6407 30.9657 38.1096 30.4969C38.5784 30.0281 39.2143 29.7647 39.8773 29.7647C40.5404 29.7647 41.1763 30.0281 41.6451 30.4969C42.114 30.9657 42.3773 31.6016 42.3773 32.2647V42.2647C42.3773 42.9277 42.114 43.5636 41.6451 44.0324C41.1763 44.5013 40.5404 44.7647 39.8773 44.7647C39.2143 44.7647 38.5784 44.5013 38.1096 44.0324C37.6407 43.5636 37.3773 42.9277 37.3773 42.2647V32.2647ZM39.8773 52.8897C39.2593 52.8897 38.6551 52.7064 38.1412 52.363C37.6273 52.0196 37.2267 51.5316 36.9902 50.9605C36.7537 50.3895 36.6918 49.7612 36.8124 49.155C36.933 48.5488 37.2306 47.992 37.6676 47.5549C38.1047 47.1179 38.6615 46.8203 39.2677 46.6997C39.8739 46.5791 40.5022 46.641 41.0732 46.8775C41.6443 47.1141 42.1323 47.5146 42.4757 48.0285C42.8191 48.5424 43.0023 49.1466 43.0023 49.7647C43.0023 50.5935 42.6731 51.3883 42.0871 51.9744C41.501 52.5604 40.7062 52.8897 39.8773 52.8897Z" fill="white"/>
+</svg>
```

Comment:
If this file is only used in `PaymentErrorView`, could it be kept inside an images folder in `./src/components/PaymentErrorView/`?